adapters/klab.ogc/src/main/java/org/integratedmodelling/klab/stac/STACEncoder.java

```diff
@@ -122,22 +125,25 @@ public void getEncodedData(IResource resource, Map<String, String> urnParameters
                     (int) grid.getXCells(), (int) grid.getYCells());
 
             Integer srid = items.get(0).getEpsg();
-            CoordinateReferenceSystem outputCrs = CrsUtilities.getCrsFromSrid(srid);
+            CrsUtilities.getCrsFromSrid(srid);
```

Comment:
Did you want to use the result of this later or is this no longer useful?

Reply:
The variable was not being used anymore, although the IDE was warning about some side effects on the method. I have made some tests have shown that both lines 127 and 128 could be deleted without affecting the final result, so I am not that sure about those side effects...